source/base/Promise.ooc

```diff
@@ -103,6 +104,13 @@ _ThreadResultPromise: class <T> extends ResultPromise<T> {
 		status := this wait()
```

Comment:
No, not really. It saves a line, but it's also much harder to read. I suppose `(this wait()) ? this _result[T] : defaultValue` doesn't work, right?